ast/pos.go

```diff
@@ -659,6 +659,23 @@ func (c *CreateRole) End() token.Pos {
 	return c.Name.End()
 }
 
+func (c *CreateChangeStream) Pos() token.Pos {
+	return c.Create
+}
+
+func (c *CreateChangeStream) End() token.Pos {
+	if len(c.Options) > 0 {
+		return c.Options[len(c.Options)-1].End() + 1
+	}
+	if len(c.Watch) > 0 {
+		return c.Watch[len(c.Watch)-1].Rparen + 1
+	}
+	if c.WatchAll {
+		return c.Name.End() + token.Pos(len("FOR ALL"))
```

Comment:
It seems incorrect to count the position by plus the `len("FOR ALL")` since there may be more than one space between FOR and ALL. Same as other places.